### ISSUE TITLE
John Mertic will connect with Jory to share with the ORES WG on standardization approaches.

### ISSUE THREAD

**Issue by chrisxie-fw** [open-renewable-energy-systems/ores#9]

**Comment by jmertic** [comment]
Howdy - do you have specific questions on standardization that I could ask Jory? Thanks in advance.

**Comment by chrisxie-fw** [comment]
Hi John,

During our first ORES meeting, you mentioned to contact Jory regarding setting up ORES as a LFESS project. There were no objections from the WG to set ORES up as an LFESS, so if you can help set up ORES in the LFESS, the issue would be resolved.

Chris

From: John Mertic ***@***.***>
Date: Friday, May 17, 2024 at 13:02
To: open-renewable-energy-systems/ores ***@***.***>
Cc: Chris Xie ***@***.***>, Author ***@***.***>
Subject: Re: [open-renewable-energy-systems/ores] John Mertic will connect with Jory to share with the ORES WG on standardization approaches. (Issue #9)

Howdy - do you have specific questions on standardization that I could ask Jory? Thanks in advance.

**Comment by jmertic** [comment]
Got it - thanks for the clarification.

@Jory Burson ***@***.***> could you add the Open Renewable
Energy Systems WG under LFESS?

Thank you,

John Mertic
Director of Program Management - Linux Foundation
Academy Software Foundation, LF Energy, and Open Mainframe Project
***@***.***
+1 234-738-4571
Schedule a meeting with me at https://meetings.hubspot.com/jmertic

**Comment by jmertic** [comment]
@chrisxie-fw is this still an open issue? If not, can you close? Thanks!

**Comment by chrisxie-fw** [comment]
Yes, this is resolved — ORES was added under LFESS (LF Energy Standards and Specifications, the JDF-based standards framework) in 2024, as reflected in the repo README and our meeting governance. Closing. Thanks for the housekeeping ping, John.
